extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts

```diff
@@ -97,23 +98,21 @@ export class PublishDatabaseDialog {
 			});
 
 			this.loadProfileButton = this.createLoadProfileButton(view);
```

Comment:
they dont get refreshed/loaded when load profile buton is clicked?

Reply:
they get loaded from the profile when the load profile button is clicked, but the "reload values from project" button reloads the values from the .sqlproj

Reply:
sounds good. just wanted to confirm that load profile refreshes (/can refresh) this table too.